ding/utils/tests/test_all_serialize.py

```diff
@@ -0,0 +1,311 @@
+#from typing import Protocol
```

Comment:
use `pytest` to construct `unittest` and `benchmark`, add translate Chinese comment to English.